src/main/java/uk/gov/hmcts/ccd/RestTemplateConfiguration.java

```diff
@@ -108,6 +109,7 @@ private HttpClient getHttpClient(final int timeout) {
                                 .useSystemProperties()
                                 .setDefaultRequestConfig(config)
                                 .setConnectionManager(cm)
+                                .setRetryHandler(new DefaultHttpRequestRetryHandler(3, false))
```

Comment:
can you explain this change?

Reply:
During performance testing of 4522 Fatih and I spotted an issue (`org.apache.http.NoHttpResponseException`) with server dropping connections under certain load and a solution on spring forums was suggested to add retry handler and also increase `http.client.validate.after.inactivity` property to `100` from intial `0` to help detect connections that have become stale (half-closed) while kept inactive in the pool.

However as this is a simple fix for only the PROD issue we could remove it from here and apply as part of 4522.

Let me know your thoughts.

Reply:
yes please let's remove it from here if they don't belong strictly to this ticket. Cheers

Reply:
Sure done.